.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: CI
+
+on:
+  pull_request:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    permissions:
+      # Allow GitHub to request an OIDC JWT ID token, for exchange with `aws-actions/configure-aws-credentials`
+      # See https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services#updating-your-github-actions-workflow
+      id-token: write
+
+      # Required for `actions/checkout`
+      contents: read
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-region: eu-west-1
+          role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+
+      - name: Setup JDK
+        uses: actions/setup-java@v3
+        with:
+          distribution: corretto
+          java-version: 11
+
+      - name: build + test
+        run: |
+          set -e
+          # Ensure we don't overwrite existing (Teamcity) builds.
+          LAST_TEAMCITY_BUILD=2360
+          export GITHUB_RUN_NUMBER=$(( $GITHUB_RUN_NUMBER + $LAST_TEAMCITY_BUILD ))
+
+          # Note, lib/test because riffraff/riffRaffUpload only tests the riffraff project.
+          # TODO add riffraff/riffRaffUpload as final sbt task below once ready to go live
+          sbt clean compile lib/test
```

Comment:
We [currently](https://github.com/guardian/riff-raff/blob/727e9d07ba0b19e640ff93d529fb8d9339e14084/script/ci) run `sbt clean compile test`. Any reason to move to `lib/test`?

Reply:
@akash1810 see the comment on L43 (a bit obscured by the temporary comment above admittedly). It's an unfortunate effect of the way this project is setup. I wonder if the multi-project lib/riffraff divide here is really serving any purpose now but wasn't keen on modifying that here.